packages/devtools_app_shared/lib/src/utils/utils.dart

```diff
@@ -146,3 +146,33 @@ extension StringUtilities on String {
     return true;
   }
 }
+
+/// Utility extension methods to the [Color] class.
+extension ColorExtension on Color {
```

Comment:
can you move this to ui/ui_utils  instead? That way we don't have to cross reference utils/ imports from the ui/theme.dart file. We try to keep things separated in this package since it is published and also for BUILD rules in g3.